README.md

```diff
@@ -100,12 +100,12 @@ npm start
 
 ### Run tests
 
-Tests under construction
+npm test
 
 
 ### Deployment
 
-Under construction
+Deployed on [Render](https://my-currency.onrender.com)
 
```

Comment:
- [Optional] Dear @Alejandro-Bernal-M, it's possible that whoever is looking at your repo, might not be a developer. They might have no idea where and how to run the `npm test` and `deploy/run` the app. I would suggest adding more clear step by step walk downs